containers/Comments/Comment.js

```diff
@@ -24,9 +24,15 @@ export const Comment = (props) => {
     ratings: {
       negative_ratings:downVotes,
       positive_ratings:upVotes,
-    }
+    },
+    id: pk,
+    comment_content_type: commentContentType,
   } = props.comment;
 
+  useEffect(() => {
+    if (props.openSubComments) {setShowSubComments(true);}
```

Comment:
suggestion: in case you want to get rid of the `if` and some `{}`, think `props.openSubComments && setShowSubComments(true);` would work too.

Reply:
To be honest, I actually prefer the more explicit notation :) But since we use the short notation everywhere, I will update it also!

Reply:
then lets go with your preferred solution! it was just a suggestion :)